client/client.go

```diff
@@ -132,7 +132,10 @@ func FromEnv(c *Client) error {
 		}
 
 		c.client = &http.Client{
-			Transport:     &http.Transport{TLSClientConfig: tlsc},
+			Transport: &http.Transport{
+				TLSClientConfig:    tlsc,
+				DisableCompression: false,
```

Comment:
This should be the default, so not needed

Reply:
I specified it to be explicit that the proper behavior of the client depends on that property value.

Also, if for whatever reason the `DisableCompression` field is removed or renamed the build would break and then things could be corrected.

Reply:
It's up to you though, do you want me to remove the setting?

Reply:
> if for whatever reason the DisableCompression field is removed or renamed

I regard the Go team highly enough that I'm confident that would never happen. I think it's safe to remove.

Reply:
Ok, I have removed this setting and rebased on master.